### DISCUSSION TITLE
Optical Music Recognition

### DISCUSSION THREAD

**Discussion by rlobet** [musescore/MuseScore#22053] [Ideas]
As developer from Université catholique de Louvain I have always been interested in pattern recognision.

Products such as 
L&H speech recognision, IRIS Data Pen Optical Character Recognision 
are part of our Louvain Technology Transfer Office 

Now I would like to know whether there has been some interest in Optical Music Recognition?

Best regards

**Comment by Jojo-Schmitz** [comment]
You may be interested in https://github.com/Audiveris/audiveris, which is what musescore.com's PDF import uses

**Comment by rlobet** [comment]
Sehr geehrter Herrn Schmitz
da wir erwarten eine sehr gute Reaktionszeiten für einen App 
die auch beim Proben von Künstler die mit einer Tablett trainieren mit Partitut von Komponisten aus den 17.-20 Jahrhundert

Meines Wissens ist MuseScore nicht für Tablette verfügbar da die MuseScore App 
Das macht die genannte Lösung unbrauchbar da sie mit Werbeanzeige für Schulungen vollgepumpt ist.

**Comment by dcorson-ticino-com** [comment]
Sehr geehrte(r) Herr/Frau rlobet,
Nach einem kurzen Studium der Merkmale des Programms wird man feststellen, dass Musescore kein Notenanzeigeprogramm ist, das von Interpreten zur Visualisierung von Stimmen verwendet wird.  Musescore ist ein Programm zur Erstellung von Druckausgaben in professioneller Qualität zusammen mit einer Tonausgabe, die inzwischen zu den besten auf dem Markt gehört.
Als Notensatzprogramm wäre eine OCR-Eingabe aus gescannten PDF-Dateien natürlich sehr interessant, um das manuelle Transkribieren vorhandener Partituren zu vermeiden.  Wenn Sie die Technologie haben, um eine fehlerfreie Noteneingabe aus einem Scan zu machen, wäre das sehr willkommen.  Alle OCR-Lösungen, die ich bisher gesehen habe, haben am Ende mehr Arbeit gemacht und länger gedauert als die Eingabe der Partitur von Hand.
Wenn Sie nach einem Notenanzeigeprogram suchen, gibt es viele auf dem Markt. Ich persönlich benutze MobileSheets, das den Vorteil hat, dass es einen extrem guten Support bietet, da das Produkt eines einzelnen Programmierers ist, der zuerst an die Qualität seines Produkts und nicht an die Finanzen denkt.  Herr Zuber von MobileSheets hat sicherlich ein offenes Ohr für alle Vorschläge, die Sie haben, und das Produkt ist für Android, Windows und Mac verfügbar.
MfG, 
Don Corson
